src/components/features/ctf-components/ctf-footer/ctf-footer.module.scss

```diff
@@ -0,0 +1,128 @@
+@use 'src/styles/utilities/breakpoint-mixins' as m;
+@import "src/styles/default-theme";
+
+.doormat_container {
+  border-top: 25px solid $secondary-color;
+  background: linear-gradient(35deg, rgba(0,0,0,1) 2%, rgba(8,71,114,1) 89%);
+}
+.doormat {
+  display: flex;
+  flex-wrap: wrap;
+  flex-flow: row wrap;
+  max-width: fit-content;
```

Comment:
In order to keep layout consistent across entire page, it's better to work with a boxed layout, so a max-width onto a media query for larger screens. For now this will do, I've added a mixin in another branch that can be reused later on to keep width consistent everywhere. Just wanted to mention this, so you're aware of this.